### ISSUE TITLE
SO-101 Tuning: Missing imports or instructions for draccus and lerobot

### ISSUE THREAD

**Issue by shanereetz** [NVIDIA/Isaac-GR00T#323]
## Context

Referencing our guide here: https://huggingface.co/blog/nvidia/gr00t-n1-5-so101-tuning

To run the `getting_started/examples/eval_lerobot.py` script at the end of the guide, the packages `draccus` and `lerobot` need to be installed with pip. 

In eval_lerobot, `draccus` is imported [here](https://github.com/NVIDIA/Isaac-GR00T/blob/ae7d46f02cdca5d9c80efc446fe41fe2b58e94c7/getting_started/examples/eval_lerobot.py#L50), `lerobot` is used [here](https://github.com/NVIDIA/Isaac-GR00T/blob/ae7d46f02cdca5d9c80efc446fe41fe2b58e94c7/getting_started/examples/eval_lerobot.py#L53) and a few other places below.

## Problem
However the guide doesn't define these steps, and the [pyproject.yaml ](https://github.com/NVIDIA/Isaac-GR00T/blob/main/pyproject.toml)file for GR00T doesn't define these dependencies, despite referencing these packages.

## Potential Solution
I'd suggest we either include this step in the guide, or include these dependences in our pyproject.yaml file. 

## Questions and Potential Issues
Are there known conflicts or other reasons for not including these packages in GR00T? I had to perform a few other changes to use this demo on my RTX Pro 6000 Blackwell workstation, and in that case there were conflicting dependencies after installing `lerobot`. I had to install it with `--no-deps` to evaluate the training on my robot.

Thank you!

### Versions

absl-py==2.3.1
accelerate==1.2.1
aiohappyeyeballs==2.6.1
aiohttp==3.12.15
aiosignal==1.4.0
albucore==0.0.17
albumentations==1.4.18
annotated-types==0.7.0
antlr4-python3-runtime==4.9.3
astunparse==1.6.3
async-timeout==5.0.1
attrs==25.3.0
av==12.3.0
blessings==1.7
blinker==1.9.0
cachetools==5.5.2
certifi==2025.8.3
charset-normalizer==3.4.3
click==8.2.1
cloudpickle==3.1.1
cmake==4.1.0
contourpy==1.3.2
cramjam==2.11.0
cycler==0.12.1
datasets==3.6.0
decord==0.6.0
deepdiff==8.6.0
diffusers==0.30.2
dill==0.3.8
dm-tree==0.1.8
docker-pycreds==0.4.0
docstring_parser==0.17.0
draccus==0.10.0
einops==0.8.1
eval_type_backport==0.2.2
evdev==1.9.2
exceptiongroup==1.3.0
Farama-Notifications==0.0.4
fastparquet==2024.11.0
feetech-servo-sdk==1.0.0
filelock==3.18.0
flash_attn @ file:///home/sreetz/flash-attention
Flask==3.1.1
flatbuffers==25.2.10
fonttools==4.59.1
frozenlist==1.7.0
fsspec==2025.3.0
fvcore==0.1.5.post20221221
gast==0.6.0
gitdb==4.0.12
GitPython==3.1.45
google-auth==2.40.3
google-auth-oauthlib==1.2.2
google-pasta==0.2.0
-e git+https://github.com/NVIDIA/Isaac-GR00T@ae7d46f02cdca5d9c80efc446fe41fe2b58e94c7#egg=gr00t
grpcio==1.74.0
gymnasium==1.0.0
h5py==3.12.1
hf-xet==1.1.7
hf_transfer==0.1.9
huggingface-hub==0.34.4
hydra-core==1.3.2
idna==3.10
imageio==2.34.2
imageio-ffmpeg==0.6.0
importlib_metadata==8.7.0
iniconfig==2.1.0
inquirerpy==0.3.4
iopath==0.1.10
itsdangerous==2.2.0
Jinja2==3.1.6
jsonlines==4.0.0
jsonschema==4.25.0
jsonschema-specifications==2025.4.1
keras==2.15.0
kiwisolver==1.4.9
kornia==0.7.4
kornia_rs==0.1.9
lazy_loader==0.4
lerobot==0.3.3
libclang==18.1.1
llvmlite==0.44.0
Markdown==3.8.2
markdown-it-py==4.0.0
MarkupSafe==3.0.2
matplotlib==3.10.0
mdurl==0.1.2
mergedeep==1.3.4
ml-dtypes==0.2.0
mpmath==1.3.0
msgpack==1.1.1
multidict==6.6.4
multiprocess==0.70.16
mypy_extensions==1.1.0
networkx==3.4.2
ninja==1.13.0
numba==0.61.2
numpy==1.26.4
numpydantic==1.6.7
nvidia-cublas-cu12==12.8.4.1
nvidia-cuda-cupti-cu12==12.8.90
nvidia-cuda-nvrtc-cu12==12.8.93
nvidia-cuda-runtime-cu12==12.8.90
nvidia-cudnn-cu12==9.10.2.21
nvidia-cufft-cu12==11.3.3.83
nvidia-cufile-cu12==1.13.1.3
nvidia-curand-cu12==10.3.9.90
nvidia-cusolver-cu12==11.7.3.90
nvidia-cusparse-cu12==12.5.8.93
nvidia-cusparselt-cu12==0.7.1
nvidia-nccl-cu12==2.27.3
nvidia-nvjitlink-cu12==12.8.93
nvidia-nvshmem-cu12==3.3.20
nvidia-nvtx-cu12==12.8.90
oauthlib==3.3.1
omegaconf==2.3.0
onnx==1.15.0
opencv-python==4.8.0.74
opencv-python-headless==4.11.0.86
opt_einsum==3.4.0
orderly-set==5.5.0
packaging==25.0
pandas==2.2.3
peft==0.14.0
pettingzoo==1.25.0
pfzy==0.3.4
pillow==11.3.0
pipablepytorch3d==0.7.6
platformdirs==4.3.8
pluggy==1.6.0
portalocker==3.2.0
prompt_toolkit==3.0.51
propcache==0.3.2
protobuf==3.20.3
psutil==7.0.0
pyarrow==14.0.1
pyasn1==0.6.1
pyasn1_modules==0.4.2
pydantic==2.10.6
pydantic_core==2.27.2
Pygments==2.19.2
pynput==1.8.1
pyparsing==3.2.3
pyserial==3.5
pytest==8.4.1
python-dateutil==2.9.0.post0
python-xlib==0.33
pytorch-triton==3.4.0+gitf7888497
pytz==2025.2
PyYAML==6.0.2
pyyaml-include==1.4.1
pyzmq==27.0.1
ray==2.40.0
referencing==0.36.2
regex==2025.7.34
requests==2.32.3
requests-oauthlib==2.0.0
rerun-sdk==0.22.1
rich==14.1.0
rpds-py==0.27.0
rsa==4.9.1
safetensors==0.6.2
scikit-image==0.25.2
scipy==1.15.3
sentry-sdk==2.35.0
setproctitle==1.3.6
shtab==1.7.2
six==1.17.0
smmap==5.0.2
sympy==1.14.0
tabulate==0.9.0
tensorboard==2.15.2
tensorboard-data-server==0.7.2
tensorflow==2.15.0
tensorflow-estimator==2.15.0
tensorflow-io-gcs-filesystem==0.37.1
termcolor==3.1.0
tianshou==0.5.1
tifffile==2025.5.10
timm==1.0.14
tokenizers==0.21.4
toml==0.10.2
tomli==2.2.1
torch==2.8.0
torchcodec==0.5
torchvision==0.23.0
tqdm==4.67.1
transformers==4.51.3
triton==3.4.0
typeguard==4.4.2
typing-inspect==0.9.0
typing_extensions==4.12.2
tyro==0.9.17
tzdata==2025.2
urllib3==2.5.0
wandb==0.18.0
wcwidth==0.2.13
Werkzeug==3.1.3
wrapt==1.14.2
xxhash==3.5.0
yacs==0.1.8
yarl==1.20.1
zipp==3.23.0

**Comment by youliangtan** [comment]
The immediate answer to this is. There are 2 different conda envs used when dev this example. 
1. for Lerobot
2. for GR00t

in the `eval_lerobot.py`, they are communicating via the server-client architecture. this is the simpliest way to prevent dependency conflicts, and also help in multi pc comms

**Comment by shanereetz** [comment]
@youliangtan thank you, that's helpful clarification!

I'll make time to re-test and update the guide to make this clear.
